recipes/urwid/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set version = "1.3.1" %}
+
+package:
+  name: urwid
+  version: "{{ version }}"
+
+source:
+  fn: urwid-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/u/urwid/urwid-{{ version }}.tar.gz
+  md5: 2e1a005cb31368fe21bfeba2d6ad5a5c
+
+build:
+  number: 0
+  skip: true  # [win]
```

Comment:
Agreed. We probably should have a few examples to demonstrate common use cases. For instance, a simple pure Python program, a C/C++ program that is maybe UNIX only, and maybe a cross platform C/C++ program (Windows too). I think these are common cases that we answer very similar questions about.

Adding the linter bot has helped some by directing people to these issues. However, you are right we should simplify for people from the outset. It will save reviewer time and leave contributors, hopefully, more satisfied.